dynatrace/api/v2/settings/objects/permissions/settings_client.go

```diff
@@ -0,0 +1,170 @@
+/*
+ * @license
+ * Copyright 2025 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package permissions
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+
+	trest "github.com/dynatrace-oss/terraform-provider-dynatrace/dynatrace/rest"
+	"github.com/dynatrace/dynatrace-configuration-as-code-core/api"
+	"github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
+)
+
+const (
+	settingsEndpoint = "platform/classic/environment-api/v2/settings"
+	schemasEndpoint  = settingsEndpoint + "/schemas"
+	objectsEndpoint  = settingsEndpoint + "/objects"
+
+	fieldsObjectID = "objectId"
+	pageSize       = 100
+)
+
+type schemaList struct {
+	Items []schemaStub `json:"items"`
+}
+
+type schemaStub struct {
+	SchemaID                string `json:"schemaId"`
+	OwnerBasedAccessControl bool   `json:"ownerBasedAccessControl"`
```

Comment:
Technically, the `OwnerBasedAccessControl` is optional. At the moment, there is only `true` and `doesn't exist at all`, but this should not matter in our case, as we fall back to false.